libraries/controllers/src/controllers/UserInputMapper.cpp

```diff
@@ -366,7 +369,7 @@ bool UserInputMapper::triggerHapticPulse(float strength, float duration, control
     Locker locker(_lock);
     bool toReturn = false;
     for (const auto& device : _registeredDevices) {
-        toReturn = toReturn || device.second->triggerHapticPulse(strength, duration, hand);
+        toReturn = device.second->triggerHapticPulse(strength, duration, hand) || toReturn;
```

Comment:
this was a logic bug we discovered a few years ago but the fix never made it in.  the old code would short-circuit after the first time toReturn was set to true, so if you had multiple devices connected, only one would buzz.  this more closely matches what the docs say this method does